### ISSUE TITLE
Adding feature in dark mode

### ISSUE THREAD

**Issue by abaran803** [NegiAkash890/editor-frontend#35]
@NegiAkash890 In dark mode, the title bar is white, which, if dark grey or black will look better.
If it is a good change to implement, please assign me.
<img width="960" alt="FireShot Capture 022 - Online Compiler - negiakash890 github io" src="https://user-images.githubusercontent.com/71405368/148284326-b9865b09-9896-4ced-a653-cad6cedd5a1a.png">

**Comment by kanthalaHarivardhanReddy** [comment]
If we set it to grey or black it will good @abaran803 
Can you assign me this issue?

**Comment by abaran803** [comment]
When I'm using this project on my local system, it is showing "Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style", a number of times.
I'm using windows 10 and vs code IDE. Not getting the solutions mentioned on StackOverflow.
![image](https://user-images.githubusercontent.com/71405368/148428259-83c83e4e-2c1c-4e43-a3d6-7400cb041530.png)

**Comment by abaran803** [comment]
> When I'm using this project on my local system, it is showing "Expected linebreaks to be 'LF' but found 'CRLF' linebreak-style", a number of times. I'm using windows 10 and vs code IDE. Not getting the solutions mentioned on StackOverflow. ![image](https://user-images.githubusercontent.com/71405368/148428259-83c83e4e-2c1c-4e43-a3d6-7400cb041530.png)

This problem is not anymore, I think this was because I did not pull the latest data. 
But now, when I am putting some js logic in NavBar.js component, it is showing an error: "Line 5:22:  Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style".
But what I know about arrow functional and react, we can put the logic inside {} and return JSX, and if we don't have any logic and there is a single block to return, we omitted the return keyword and {}.

**Comment by NegiAkash890** [comment]
Can you share a screenshot so I can have a better idea regarding the same?

**Comment by abaran803** [comment]
> Can you share a screenshot so I can have a better idea regarding the same?
![Screenshot (153)](https://user-images.githubusercontent.com/71405368/148507842-3ac40b14-396f-4127-b7eb-ca35050bde29.png)
Returning a single component without return keyword and without curly braces, working fine


![Screenshot (154)](https://user-images.githubusercontent.com/71405368/148507923-3125edc1-f313-4b8d-9206-8d30baefa40c.png)
Returning the same component with the return keyword and curly braces, not working. I think, to put some js code, we require to use the different blocks of code, one is the logic part and the other is JSX.

Please accept my LinkedIn connection request, I want to discuss and learn more about the project and its structure.

**Comment by NegiAkash890** [comment]
Hi @abaran803 Just replace 
```javascript 
const NavBar = () => {}
```

with

```javascript  
const NavBar = () => ( );     //Also remove the return statement before div
```

**Comment by abaran803** [comment]
> Hi @abaran803 Just replace
> 
> ```js
> const NavBar = () => {}
> ```
> 
> with
> 
> ```js
> const NavBar = () => ( );     //Also remove the return statement before div
> ```

Then how can I write the logic for this component?

**Comment by NegiAkash890** [comment]
> > Hi @abaran803 Just replace
> > ```js
> > const NavBar = () => {}
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > with
> > ```js
> > const NavBar = () => ( );     //Also remove the return statement before div
> > ```
> 
> Then how can I write the logic for this component?

Try this !

```javascript
const NavBar = () => {
  // logic Here
  const [name, setName] = useState('Akash');
}

```

**Comment by abaran803** [comment]
Again same problem
![image](https://user-images.githubusercontent.com/71405368/148829898-d13ff8e8-6a15-4e16-9f9e-9eca1027b851.png)
![image](https://user-images.githubusercontent.com/71405368/148829914-dbe5f628-2623-4fb1-8b11-11b3f296b77f.png)

**Comment by NegiAkash890** [comment]
> Again same problem ![image](https://user-images.githubusercontent.com/71405368/148829898-d13ff8e8-6a15-4e16-9f9e-9eca1027b851.png) ![image](https://user-images.githubusercontent.com/71405368/148829914-dbe5f628-2623-4fb1-8b11-11b3f296b77f.png)

That's because of `ESLINT`. If we have some variables that aren't being used then it throws this error. 
You can read more about this here : https://eslint.org/docs/user-guide/getting-started

For now just add this
```
 /* eslint-disable */
```
at the top of the file

**Comment by NegiAkash890** [comment]
Closed via #38
